ejb3/src/main/java/org/jboss/as/ejb3/timerservice/TimerServiceImpl.java

```diff
@@ -257,13 +260,13 @@ public Timer createIntervalTimer(Date initialExpiration, long intervalDuration,
             throws IllegalArgumentException, IllegalStateException, EJBException {
         assertTimerServiceState();
         if (initialExpiration == null) {
-            throw EJB3_TIMER_LOGGER.initialExpirationIsNullCreatingTimer();
+            throw EJB3_TIMER_LOGGER.invalidTimerParameter("initialExpiration", null);
         }
         if (initialExpiration.getTime() < 0) {
```

Comment:
You might consider [org.wildfly.common.Assert.checkMinimumParameter](https://github.com/wildfly/wildfly-common/blob/1.5.4.Final/src/main/java/org/wildfly/common/Assert.java#L305) for these cases.